src/test/java/com/users/utility/UserUtilityTest.java

```diff
@@ -0,0 +1,69 @@
+package com.users.utility;
+
+import com.users.dto.UserDto;
+import com.users.entities.User;
+import org.testcontainers.shaded.org.apache.commons.lang3.RandomStringUtils;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class UserUtilityTest {
+
+    private static final String mail = "@yahoo.com";
```

Comment:
prorietatile care sunt final se scriu cu litere mari, asa e recomandat gen String MAIL =..ca sa sara mai usor in ochi ca e final